windows/wsl_distro_install_uninstall/install_wsl.yml

```diff
@@ -41,6 +70,7 @@
   when:
     - win_powershell_cmd_output.rc is defined
     - win_powershell_cmd_output.rc != 0
+    - not skip_set_wsl_ver
```

Comment:
Where is the default value of skip_set_wsl_ver set? I only saw it's set as true when `guest_os_build_num | int <= 23048` and `guest_os_product_type | lower == 'server'`. If it is not initialized, you'd better to add checking about skip_set_wsl_ver is defined or not.